ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Cli.scala

```diff
@@ -26,30 +30,49 @@ object Cli {
         },
       opt[Config.StreamConfig]("consume-stream")
         .abbr("s")
+        .unbounded()
+        .minOccurs(1)
         .text(
           s"Stream configuration."
         )
-        .valueName("streamType=<transactions|transaction-trees>,name=<streamName>,party=<party>")
-        .action { case (streamConfig, config) => config.copy(streamConfig = Some(streamConfig)) },
+        .valueName(
+          "stream-type=<transactions|transaction-trees>,name=<streamName>,party=<party>[,begin-offset=<offset>][,end-offset=<offset>][,template-ids=<id1>|<id2>]"
+        )
+        .action { case (streamConfig, config) =>
+          config.copy(streams = config.streams :+ streamConfig)
+        },
+      opt[FiniteDuration]("log-interval")
+        .abbr("r")
+        .text("Stream metrics log interval.")
+        .action { case (period, config) => config.copy(reportingPeriod = period) },
+      opt[Int]("core-pool-size")
+        .text("corePoolSize parameter of the ThreadPoolExecutor.")
```

Comment:
users are not interested in such an implementation detail
```suggestion
        .text("Size of the worker thread pool.")
```